modules/acquia_cms_headless/tests/src/Functional/SitePreviewTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Drupal\Tests\acquia_cms_headless\Functional;
+
+use Acquia\DrupalEnvironmentDetector\AcquiaDrupalEnvironmentDetector;
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+use Drupal\next\Entity\NextEntityTypeConfig;
+
+/**
+ * Tests headless content site preview.
+ *
+ * @group acquia_cms
+ * @group acquia_cms_headless
+ * @group medium_risk
+ * @group push
+ */
+class SitePreviewTest extends WebDriverTestBase {
```

Comment:
Please use HeadlessTestBase.

@apathak18

Reply:
@deepakmishra2 Not required as I have to login via admin and created a test content type in setup.